src/scale/Log.ts

```diff
@@ -36,9 +36,28 @@ const roundingErrorFix = numberUtil.round;
 const mathFloor = Math.floor;
 const mathCeil = Math.ceil;
 const mathPow = Math.pow;
-
 const mathLog = Math.log;
 
+/**
+ * symmetric log, allowing negative values for logarithmic scales
+ */
+function symMathLog(x: number): number {
+    if (x >= 0) {
+        return mathLog(x);
+    }
+    return -mathLog(-x);
+}
+
+/**
+ * symmetric pow, allowing negative values for logarithmic scales
+ */
+function symMathPow(x: number, y: number): number {
+    if (y >= 0) {
```

Comment:
Suggestion for slightly more terse syntax:

```
return y >= 0 ? mathPow(x, y) : -mathPow(x, -y);
```

Reply:
Good idea!